### DISCUSSION TITLE
Private rooms

### DISCUSSION THREAD

**Discussion by lmtr0** [ONLYOFFICE/DocumentServer#1993] [Q&A]
Hello there, good day! I have a few questions about private rooms. I'm building an end-to-end encrypted app that will use OnlyOffice for document edition, and I plan on using private rooms for that.

Can I use private rooms with my storage service? Can It run in the browser? Can I use private rooms only for editing files (meaning, can I create, move files and use private rooms only for editing it's contents)?

**Comment by Gingetsu** [comment]
Hello, @lmtr0 ! At the moment, private rooms works only in the desktop editor. Yes, you will be able to edit documents. While editing the document, you will be able to save it to yourself.

**Reply by lmtr0** [reply]
Hi there, thank you for the reply, can you please share some documentation with me pointing on how to configure document server for that to work? Is there any way to make it run in the browser? With libsodium.js for example?
